source/docs/software/driverstation/driver-station-best-practices.rst

```diff
@@ -1,77 +1,97 @@
-6. Change "Active Hours" for Windows Updates to prevent updates from installing during competition hours. Navigate to Start -> Settings -> Update & Security -> Windows Update, then select Change active hours. If you’re traveling to a competition, take time zone differences into account. This will help ensure your driver station does not reboot or fail due to update installing on the field.
-7. Remove any 3rd party antivirus or antimalware software. Instead, use Windows Defender on Windows 10 or 11. Since you're only connecting to the internet for Windows and FRC software updating, the risk is low. Only install software on your driver station that's needed for driving. Your goal here is to eliminate variables that might interfere with proper operation. Remove any unneeded preinstalled software ("bloatware") that came with the machine. Don't use the laptop as your Steam machine for gaming back at the hotel the night before the event. Many teams go as far as having a separate programming laptop.
-8. Avoid managed Windows 10 or 11 installations from the school's IT department. These deployments are built for the school environment and often come with unwanted software that interferes with your robot's operation.
-9. Laptop battery / power
+#.  A keyboard. It’s hard to quickly do troubleshooting on touch-only computers at the field.
+#.  A solid-state disk (SSD), 256 GB or larger. If the laptop has a rotating disk, spend $50 and replace it with a SSD.
+#.  Supports Windows 11.  While Windows 10 is still supported for driver station use, acquiring a machine that can't run Windows 11 isn't forward thinking.
+#.  AMD or Intel CPU. The Driver Station software does not work on machines with ARM CPUs.
+#.  2.4 and 5 GHz Wi-Fi radio, with 6GHz optional.  6GHz Wi-Fi can be used in some of the practice driving configurations.   See `recommended practice configuration <https://frc-radio.vivid-hosting.net/overview/practicing-at-home>`__ for details.
 
-   a. Turn off Put the computer to sleep in your power plan for both battery and powered operation.
-   b. Turn off Battery Saver and set Power Mode to Best Performance.
-   c. Turn off USB Selective Suspend:
+## Building your driver station
 
-      i. Right click on the battery/charging icon in the tray, then select Power Options.
-      ii. Edit the plan settings of your power plan.
-      iii. Click the Change advanced power settings link.
-      iv. Scroll down in the advanced settings and disable the USB selective suspend setting for both Battery and Plugged in.
+It's best if you combine your driver station laptop and all needed peripherals into a `single unit <https://www.google.com/search?udm=2&q=frc+driver+station>`__.
```

Comment:
I don't really like linking to a google search that can change over time. Maybe a CD thread like https://www.chiefdelphi.com/t/looking-for-driver-station-ideas/463986 or https://www.chiefdelphi.com/t/show-me-your-operator-station-driver-station/435030 although neither have as many examples as I'd like.